### DISCUSSION TITLE
Speech option to create list of items

### DISCUSSION THREAD

**Discussion by kmarbach** [DelphiWorlds/Kastri#205] [Ideas]
With something I'm doing I'd like a user to be able to list several things (people's names), and to configure the speech recognition to treat a 1 sec delay (configurable) as basically a hard return, or any sort of delimiter. Is that possible, or can it be added?

**Comment by DelphiWorlds** [comment]
Is there some reason you cannot handle this yourself in the event that returns the text of the speech?

**Comment by kmarbach** [comment]
I will see if I can – I haven’t used it in a year and am starting a new project, new computer, and having trouble getting things to work again. From memory, when user stops speaking is when the recording stops. User would need to say a delimiter so I can parse to determine each line, correct? What I was hoping was a pause of, say, ¾ seconds, indicates end of item, pause of 1.5 seconds indicates end of list.

Keith
